var/spack/repos/builtin/packages/tau/package.py

```diff
@@ -148,6 +153,10 @@ def install(self, spec, prefix):
         if '+binutils' in spec:
             options.append("-bfd=%s" % spec['binutils'].prefix)
 
+        if '+libdwarf' in spec:
+            options.append("-dwarf=%s" % spec['libdwarf'].prefix)
+            options.append("-elf=%s" % spec['libelf'].prefix)
```

Comment:
This should be in a different `if` statement. Just because `+libdwarf` is activated doesn't mean `+libelf` is too.